scripts/steps/build-api.js

```diff
@@ -110,6 +104,111 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'Connection',
+  'Log',
+  'MemoryStore',
+  'Reconnect',
+  'ServerConnection',
+  'defineAction',
+  'isFirstOlder',
+  'parseId'
+])
+const CLIENT_ENTITIES = new Set([
+  'Client',
+  'CrossTabClient',
+  'IndexedStore',
+  'attention',
+  'badge',
+  'buildNewSyncMap',
+  'changeSyncMap',
+  'changeSyncMapById',
+  'confirm',
+  'createFilter',
+  'createSyncMap',
+  'defineSyncMap',
+  'deleteSyncMap',
+  'deleteSyncMapById',
+  'encryptActions',
+  'favicon',
+  'log',
+  'request'
+])
+const SERVER_ENTITIES = new Set([
+  'ChannelContext',
+  'Context',
+  'ResponseError',
+  'Server',
+  'ServerClient',
+  'del',
+  'get',
+  'patch',
+  'post',
+  'put',
+  'request'
+])
+const STATE_ENTITIES = new Set([
+  'createDerived',
+  'createMap',
+  'createPersistent',
+  'createRouter',
+  'createStore',
+  'getPagePath',
+  'getValue',
+  'openPage'
+])
+const TEST_ENTITIES = new Set([
+  'TestClient',
+  'TestLog',
+  'TestPair',
+  'TestServer',
+  'TestTime',
+  'cleanStores',
+  'emptyInTest',
+  'prepareForTest'
+])
+
+const GROUPS = {
+  Client: 'Client',
+  Core: 'Core',
+  Other: 'Other',
+  Preact: 'Preact',
+  React: 'React',
+  Server: 'Server',
+  State: 'Logux State',
+  Tests: 'Tests',
+  Vue: 'Vue'
+}
+
+// More specific group conditions should be placed higher
+const GROUPS_CONDITION = {
+  [GROUPS.React]: node => /^react\./i.test(node.name),
```

Comment:
I decide to change condition to framework group to Regexp since all method and classes have a framework prefix in name